CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/parameters/SelectOptionsExtension.java

```diff
@@ -0,0 +1,98 @@
+/*
+* Copyright 2010-2023 Australian Signals Directorate
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.parameters;
```

Comment:
If this is being used by more than just multi choice plugin parameters, I wonder whether Core Utilities may be a better place to put this?

Reply:
Early on in developing a solution I explored using Core Utilities to house the bulk selection functionality but believe I came across issues with conflicting dependencies with whatever class I was moving at the time. I have just checked with this final solution, and it is happy to live in Core Utilites so will move it as I agree with your suggestion.